commissioning.tex

```diff
@@ -1,27 +1,50 @@
+
 \subsection{Commissioning Schedule}
 \label{ssec:commissioning-schedule}
 
-In December 2022, as a result of reoptimizing the sequence of integration activities, the Rubin Construction Project decided to install the LSST Science Camera (LSSTCam) on the telescope earlier in the assembly sequence than previously planned. 
-As a consequence, no on-sky data will be taken with the Commissioning Camera (ComCam). 
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is reoptimizing the sequence of integration activities based on important updates to projected subcomponent availability.
+Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
+This will enable Telescope commissioning, primarily the Active Optics System, to occur approximately four months earlier than could be done with LSSTCam. 
+With this updated strategy, System First Light with LSSTCam is projected to occur in January 2025, with Construction completion in May 2025.
+Without these changes, we predict an additional two months of delay.
 
 \begin{figure}[htb]
 \centering
-\includegraphics[width=0.95\linewidth]{figures/rubinobs_on-sky_commissioning_gantt.pdf}
-\caption{Detailed schedule of commissioning activities relative to System First Light, as of December 2022.}
-\label{fig:commissioning-gantt}
+\includegraphics[width=0.98\linewidth]{figures/rubinobs_on-sky_commissioning_and_early_science.pdf}
+\caption{Detailed schedule of commissioning  and early science activities relative to System First Light, as of October 2023.}
+\label{fig:commissioning-es-schedule}
+\vspace{0.1cm}
 \end{figure}
 
-Figure~\ref{fig:commissioning-gantt} shows the detailed schedule of commissioning activities relative to System First Light, as of December 2022.
-System First Light is currently expected between October 2024 and February 2025 (\S~\ref{sec:timeline}), about 3 months after First Photon. 
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
 The total amount of science validation time currently planned is about 8 weeks.  
 LSST data taking is expected to start about 4 months after System First Light.
```

Comment:
With schedule uncertainty. LSST is 4-10 months after first light.

"LSST data taking is expected to start 4-10 months after System First Light depending on construction schedue uncertainty and Operations readiness to start the survey following the start of full Survey Operations.